extra/man/alacritty.5.scd

```diff
@@ -195,6 +195,12 @@ This section documents the *[window]* table of the configuration file.
 
 	Default: _"None"_
 
+*level* = _"Normal"_ | _"AlwaysOnTop"_
+
+	Sets the window level (Normal, AlwaysOnTop).
+
+	Default: _"Normal"_
```

Comment:
The different window levels should be documented, see `startup_mode` for reference.

Having the variants in parenthesis after the description is not beneficial, so that should be removed.